### ISSUE TITLE
Does every test sample need a corresponding sparse depth map to improve testing accuracy?

### ISSUE THREAD

**Issue by sconlyshootery** [zswang666/PnP-Depth#2]
It seems that every z is generated from a particular test sample x, we need ground truth sparse depth map D_s to update z for each x. Does it mean that during testing, every test sample x need corresponding ground truth sparse depth map D_s to update corresponding z?

**Comment by zswang666** [comment]
correct, pnp-depth is designed for task with access of sparse ground truth, e.g., depth completion. Also, in such sense, the exactly same technique can be applied to other task than depth estimation as long as you have partial ground truth!
